src/nvim/tui/tui.c

```diff
@@ -386,7 +386,7 @@ static void terminfo_start(TUIData *tui)
 
   char *term = os_getenv("TERM");
 #ifdef MSWIN
-  os_tty_guess_term(&term, tui->out_fd);
+  os_tty_guess_term((const char **)&term, tui->out_fd);
```

Comment:
We should probably change the retval of `os_getenv` to be `const`. This is fine for now though.